.github/workflows/unit_tests.yml

```diff
@@ -14,26 +14,15 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        include:
-         - os: ubuntu-latest
-           archs: x86_64 i686
-         #- os: arm-4core-linux
-         #  archs: aarch64
-         - os: windows-latest
-           archs: AMD64 x86
-         - os: macos-latest
-           archs: arm64
+        os: [ubuntu-latest, windows-latest, macos-latest]
+        # Keep this in sync with hatch.toml
+        python-version: ["3.7", "3.10", "3.12"]
     steps:
       - uses: actions/checkout@v4
         # Include all history and tags
         with:
           fetch-depth: 0
 
-      - uses: actions/setup-python@v5
```

Comment:
We don't need this anymore because hatch is fancy and will download the right Python for us?

Reply:
Yes, as far as I know, hatch tries to use system python if there's one with requested version, and installs one if it can't find.